src/views/Home/Home.jsx

```diff
@@ -0,0 +1,23 @@
+import './Home.css';
+import { Link } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+
+export default function Home() {
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
```

Comment:
Not entirely sure what this is doing, looks as if it sets loading false immediately